<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLConcludeClass</class>
 <widget class="QWidget" name="DLConcludeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>667</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>深度学习应用</string>
  </property>
  <widget class="QLabel" name="show_img">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>640</width>
     <height>640</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>640</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>这里预览图像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>661</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>861</width>
     <height>641</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>861</width>
     <height>999</height>
    </size>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>人体检测</string>
    </attribute>
    <widget class="QPushButton" name="upload_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>上传图像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_camera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>打开摄像头</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>841</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>信息统计</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QTableWidget" name="info_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>811</width>
        <height>441</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>类别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>中心位置:X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>中心位置:Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>置信度</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="save_img">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>保存当前图像</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>61</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>置信度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="confidence">
        <property name="minimum">
         <double>0.300000000000000</double>
        </property>
        <property name="maximum">
         <double>0.950000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>超分辨率+无损缩放</string>
    </attribute>
    <widget class="QPushButton" name="upload_img_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>上传图像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_img_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>保存当前图像</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>351</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="resize_op">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>缩放操作</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="width">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1920</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1920</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="height">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1080</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1080</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="keep_radio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>保持纵横比</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>表情识别</string>
    </attribute>
    <widget class="QPushButton" name="upload_img_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>上传图像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_camera_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>打开摄像头</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>61</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>置信度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="confidence_4">
        <property name="minimum">
         <double>0.300000000000000</double>
        </property>
        <property name="maximum">
         <double>0.950000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="save_img_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>保存当前图像</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>841</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>信息统计</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QTableWidget" name="info_table_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>811</width>
        <height>441</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>类别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>表情</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>置信度</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>人体分割</string>
    </attribute>
    <widget class="QPushButton" name="open_camera_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>打开摄像头</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_background">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>添加背景</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_background">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>清空背景</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>待开发</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
